INSTALL.md

```diff
@@ -0,0 +1,95 @@
+# Install Tomatoes
+
+## Clone it!
+Of course, you will need a local copy of this repo. Just clone it:
+```sh
+git clone https://github.com/tomatoes-app/tomatoes
```

Comment:
I chose HTTPS because of this one: http://stackoverflow.com/a/11041782
Atleast GitHub suggests it too: https://help.github.com/articles/which-remote-url-should-i-use/
Don't know. Think that's a trivial point.